backend/src/server.js

```diff
@@ -1,13 +1,18 @@
-const express = require('express');
-const cors = require('cors');
-const path = require('path');
-const morgan = require('morgan');
-const authRouter = require('./api/auth/auth.routers');
-const usersRouter = require('./api/users/user.router');
-const transactionRouter = require('./api/transactions/transactionRouter');
-require('dotenv').config({ path: path.join('./.env') });
-
-const AppError = require('./api/errors/appError');
+const express = require("express");
+const cors = require("cors");
```

Comment:
Prettier? Why we have different quotes here?

Reply:
ok